lcm/lcm-cpp-impl.hpp

```diff
@@ -35,7 +35,7 @@ class LCMTypedSubscription : public Subscription {
             return;
         }
         const ReceiveBuffer rb = {rbuf->data, rbuf->data_size, rbuf->recv_utime};
-        subs->handler(&rb, channel, &msg, subs->context);
+        subs->handler(&rb, subs->channel(channel), &msg, subs->context);
```

Comment:
This seems like it shouldn't compile, as std::string doesn't define `opeator()`. Any reason this isn't preceded by an assignment like the others? Am I missing something?